draft-ietf-suit-architecture.md

```diff
@@ -300,7 +300,7 @@ The following stakeholders are used in this document:
 
 * Device Operator: The device operator is responsible for the day-to-day operation
   of a fleet of IoT devices. Customers of IoT devices, as the owners of
-  IoT devices - such as enterprise customers or end users, interact
+  IoT devices - such as enterprise customers or end users - interact
```

Comment:
This one I'm not sure I had the proper fix for -- there seem to be a lot of parenthetical phrases in here and I'm not sure how they're intended to fit together.

Reply:
This looks right to me.